101-vm-from-user-image-and-datadisk/README.md

```diff
@@ -0,0 +1,25 @@
+# Create a Virtual Machine from a User Image
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fiandow%2Fazure-quickstart-templates%2Fmaster%2F101-vm-from-user-image-and-datadisk%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+Prerequisite - The Storage Account with the User Image VHD should already exist in the same resource group.
+
+This template allows you to create a Virtual Machines from user specified images for OS and Data disks. This template also deploys a Virtual Network, Public IP addresses and a Network Interface.
+
+Below are the parameters that the template expects
+
+| Name   | Description    |
```

Comment:
@singhkay I removed the table and pushed that change back to my master branch. Do I need to resubmit this PR?